main.go

```diff
@@ -96,6 +96,8 @@ func getLog(
 	// intentions...
 	writer.Header().Set("Content-Type", "text/plain; charset=utf-8")
 	writer.Header().Set("Access-Control-Allow-Origin", "*")
+	writer.Header().Set("Transfer-Encoding", "chunked")
```

Comment:
Looks like that might be done [here](https://github.com/golang/go/blob/b992c391d4aae64e147fc64c77ad41d61be8e2e7/src/net/http/transfer.go#L72-L74).